pkg/target/manager.go

```diff
@@ -307,11 +304,11 @@ func (m *managerImpl) TargetShoot(ctx context.Context, shootName string) error {
 		} else if t.Seed != "" {
 			seed, err = m.resolveSeedName(ctx, gardenClient, t.Seed)
 			if err != nil {
-				return fmt.Errorf("failed to fetch kubeconfig for seed cluster: %w", err)
+				return fmt.Errorf("failed to resolve sed by name: %w", err)
```

Comment:
```suggestion
				return fmt.Errorf("failed to resolve seed by name: %w", err)
```

Reply:
however this error message is not consistent with the error message for `resolveShootName`, which is `failed to resolve shoot`